### ISSUE TITLE
slack integration

### ISSUE THREAD

**Issue by tansaku** [AgileVentures/AsyncVoter#12]
we imagine from within a project channel starting a vote like this:

> @async_bot start vote on http://ticket/id

then bot would respond with vote instructions in channel

> vote on http://ticket/id "title", please DM 1(simple), 2 (medium), 3(hard) to me at @async_bot - please discuss here in channel or in ticket as preferred

then individuals DM bot and bot updates channel with names of voters, revealing the vote values when we reach a threshold of, say 3 votes

**Comment by krauszraphael** [comment]
Wouldn't it be easier for the bot, just to parse the ticket id:

e.g. @async_bot http://ticket/id 2

**Comment by tansaku** [comment]
yes indeed - should be stripped to bare minimum - actually I'm thinking that tracer bullet needs to be done here - we need to get something working that interacts with slack so we can get the user experience down ...

**Comment by tansaku** [comment]
I should grab a node slack lib and just run my manual async votes but do it through the slack API: https://www.npmjs.com/package/slack

**Comment by tansaku** [comment]
ultimately want https://api.slack.com/bot-users
